### DISCUSSION TITLE
Deceptive Site Warning on My Simple URL Shortening App: https://shorty.deta.dev/

### DISCUSSION THREAD

**Discussion by dhanushreddy291** [deta/cloud-docs#253] [Help] [answered]
Can anyone help me how to fix Deceptive Site ahead for my App: [https://shorty.deta.dev/](https://shorty.deta.dev/)

Basically, it is a very simple URL Shortner. I don't know why Google Chrome is saying like this. It worked fine for a week and then this all of a sudden.

**Comment by abdelhai** [comment, marked as answer]
hi, i've tested on chrome and didn't get any errors. google's official safe browsing tool isn't showign any issues: https://transparencyreport.google.com/safe-browsing/search?url=https:%2F%2Fshorty.deta.dev%2F&hl=en

make sure you are on the latest version of chrome. let me know if the issue persists or any further details that could help us narrow this down

**Reply by dhanushreddy291** [reply]
Really, it showed warning before, even today morning. I have no idea how it went off now magically.

Thank you so much again for testing. 😄
